### ISSUE TITLE
Review decision about using "px" or "rem" in Dropdown component min/max-width declarations

### ISSUE THREAD

**Issue by didoo** [hashicorp/design-system#228]
We have decided to defer the discussion and decision about using `px` or `rem` in the `Dropdown` component `min/max-width` declarations to a new issue, to unblock the merging of the other PRs related to the `Dropdown`.
https://github.com/hashicorp/design-system/pull/202#discussion_r852143568

/cc @MelSumner @Dhaulagiri

**Comment by MelSumner** [comment]
Should we move this to our backlog in JIRA?
